src/components/pricing/email-form.tsx

```diff
@@ -0,0 +1,128 @@
+import stripeCheckoutRedirect from 'api/stripe/stripe-checkout-redirect'
+import {Formik} from 'formik'
+import * as React from 'react'
+import ReactMarkdown from 'react-markdown'
+import axios from 'utils/configured-axios'
+import * as yup from 'yup'
+
+const loginSchema = yup.object().shape({
+  email: yup.string().email().required('enter your email'),
+})
+
+type EmailFormProps = {
+  priceId: string
+}
+
+const EmailForm: React.FunctionComponent<EmailFormProps> = ({priceId}) => {
+  const [isSubmitted, setIsSubmitted] = React.useState(false)
+  const [isError, setIsError] = React.useState<boolean | string>(false)
+
+  const validateEmail = async (email: string) => {
+    const isPro = await axios
+      .post(`/api/users/check-pro-status`, {
+        email,
+      })
+      .then(({data}) => data)
+
+    console.log(isPro)
+
+    if (isPro) {
```

Comment:
this is a quick and dirty lookup to see if we already have a subscription on this particular email

not great UX